README.md

```diff
@@ -290,6 +290,25 @@ The most commonly adjusted settings are probably
 
 Also, you can add additional vocabulary words, words to replace, search engines and more. Complete the community setup instructions above, then open the `settings` folder to see the provided CSV files and customize them as needed.
 
+## Overriding settings without forking
```

Comment:
```suggestion
## 💡 Tip: Overriding cleanly
```

Reply:
I think we should probably focus on describing the benefits in simple terms (which I think for most people is "how do I make changes in the best possible way?" / "how do I reduce the pain of merging with community?" which is absolutely something people run into).

I think it's great that you're raising the need to reduce forking, but I don't think that philosophy is something the average new user would readily be familiar with (keep in mind a decent number of users aren't familiar with git and start by downloading the repository as a .zip), and leading with "without forking" sort of assumes that you already know why that's valuable. But making changes that won't be a pain to merge in the future is something almost all of our users have to deal with, so I would personally lead with that.

I think if we get more content about making changes cleanly/sustainably, we could probably break that out to a dedicated page/article, and from there get into depth about the philosophy of forking community, for people who are interested.

(As as aside, I wish we had a better term to capture the classical OSS meaning of the word "fork". GitHub has blurred the term significantly, since they encourage everyone to make a fork in order to open a pull request, and it's a painless one click operation that isn't an obviously bad thing. I think this makes it more difficult to talk about why "forking" is a bad thing, because some people will just think you mean creating your own copy (necessarily, sometimes!) of the repository on the web. "hard fork" is the closest term I can think of that comes closest to the original meeting, but I'd be interested in if there are any others. This will probably help your broader discussion goals by avoiding this ambiguity.)

Reply:
I'm fine with changing the wording to emphasize the benefits you called out here. The target audience I had in mind is not the average talon user but rather someone coming to talon who might be open / interested in contributing to community. Those folks would likely already be familiar with the benefits of avoiding forking but wouldn't know how to manage the configuration necessary to do so without struggling (as I did) because of the complexity involved in configuring community without editing the repository.